docs/the_bat_header_format.md

```diff
@@ -5,17 +5,28 @@ tags:
 ---
 The Bat! v3
 
-    Date: Mon, 6 Aug 2007 00:41:44 +0200
+    Date: Fri, 7 Feb 2025 15:23:52 +0300
     From: Username <username@sendinghost.com>
-    X-Mailer: The Bat! (v3.95.8) Professional
-    X-Priority: 3 (Normal)
-    Message-ID: <1398886086.20070806004144@sendinghost.com>
+    Message-ID: <1066947962.20250207152352@gmail.com>
     To: Username <username@receivinghost.com>
     Subject: header test. the bat
     MIME-Version: 1.0
     Content-Type: text/plain; charset=us-ascii
     Content-Transfer-Encoding: 7bit
+    Content-Type: multipart/mixed; boundary="----------069134143049E2085"
 
+Message-ID format:
+
+`Message-ID` consists of 4 parts
+
+* A random string of numbers
+* A timestamp of the format YYYYmmDDHHMMSS
+* The `@` symbol
+* The domain of the sending host
+
+As at March 2025, there is [an interesting nuance](https://www.bentasker.co.uk/posts/blog/security/seducing-a-romance-scammer.html#scheduled-sending) in `Message-ID` when used with the mail agent's [Postponed Sending](https://www.ritlabs.com/en/products/thebat/postponed.php) feature. When a mail's sending is postponed, the `Date` header will be updated accordingly, but the timestamp within the Message ID _will not_.
```

Comment:
More than happy to remove this link if it's too cheeky :)

Reply:
LGTM!